python/py-importlib-resources/Portfile

```diff
@@ -33,20 +33,39 @@ if {${name} ne ${subport}} {
                     port:py${python.version}-toml
 
     if {${python.version} == 27} {
+        version     3.3.1
+        revision    0
+        checksums   rmd160  6b0e6b482a5cb45430243e78317731efcf7eb84b \
+                    sha256  0ed250dbd291947d1a298e89f39afcc477d5a6624770503034b72588601bcc05 \
+                    size    29012
+
         depends_lib-append \
                     port:py${python.version}-contextlib2 \
                     port:py${python.version}-pathlib2 \
-                    port:py${python.version}-singledispatch
-    }
-
-    if {${python.version} < 35} {
+                    port:py${python.version}-singledispatch \
+                    port:py${python.version}-typing
+    } elseif {${python.version} == 34} {
+        version     1.0.2
+        epoch       1
+        revision    0
+        checksums   rmd160  ccbed771125ab62eda228842bc87339dd9aa21a8 \
+                    sha256  d3279fd0f6f847cced9f7acc19bd3e5df54d34f93a2e7bb5f238f81545787078 \
+                    size    23602
+        depends_build-append \
+                    port:py${python.version}-wheel
         depends_lib-append \
+                    port:py${python.version}-pathlib2 \
                     port:py${python.version}-typing
+    } elseif {${python.version} == 35} {
+        version     3.2.1
+        revision    0
+        checksums   rmd160  5abb9e9bbae805089609dca64868d2ea7a8629fd \
+                    sha256  a9fe213ab6452708ec1b3f4ec6f2881b8ab3645cb4e5efb7fea2bbf05a91db3b \
+                    size    28884
     }
 
     if {${python.version} < 38} {
         depends_lib-append \
-                    port:py${python.version}-importlib-metadata \
```

Comment:
Are you sure this is not needed anymore?

Reply:
Yes, you can check upstream's release notes and/or `setup.cfg` - the dependency it was removed in version 3.x ; and version 1.0.2 (for PY34) didn't require it either.